modules/withRouter.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react'
 import hoistStatics from 'hoist-non-react-statics'
 import { routerShape } from './PropTypes'
+import warning from './routerWarning'
+
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component'
 }
 
-export default function withRouter(WrappedComponent) {
+export default function withRouter(WrappedComponent, options) {
```

Comment:
That'd require doing e.g. `{ withRef: false } = {}`, though. The transpiled code for default params is a bit ugly, plus there's that extra temporary object.